dashboard/app/models/blockly.rb

```diff
@@ -165,124 +186,15 @@ def blockly_options
-        nectar_goal
-        honey_goal
-        flower_type
-        skip_instructions_popup
-        is_k1
-        required_blocks:levelBuilderRequiredBlocks
-        toolbox_blocks:toolbox
-        x:initialX
-        y:initialY
-        maze:map
-        ani_gif_url:aniGifURL
-        shapeways_url
-        images
-        free_play
-        min_workspace_height
-        permitted_errors
-        disable_param_editing
-        disable_variable_editing
-        success_condition:fn_successCondition
-        failure_condition:fn_failureCondition
-        first_sprite_index
-        protaganist_sprite_index
-        timeout_failure_tick
-        soft_buttons
-        edge_collisions
-        projectile_collisions
-        allow_sprites_outside_playspace
-        sprites_hidden_to_start
-        background
-        coordinate_grid_background
-        use_modal_function_editor
-        use_contract_editor
-        contract_highlight
-        contract_collapse
-        examples_highlight
-        examples_collapse
-        definition_highlight
-        definition_collapse
-        disable_examples
-        default_num_example_blocks
-        impressive
-        open_function_definition
-        disable_sharing
-        edit_code
-        code_functions
-        app_width
-        app_height
-        embed
-        generate_function_pass_blocks
-        timeout_after_when_run
-        custom_game_type
-        project_template_level_name
-        scrollbars
-        is_project_level
-        failure_message_override
-        show_clients_in_lobby
-        show_routers_in_lobby
-        can_connect_to_clients
-        can_connect_to_routers
-        show_add_router_button
-        message_granularity
-        automatic_receive
-        router_expects_packet_header
-        client_initial_packet_header
-        show_add_packet_button
-        show_packet_size_control
-        default_packet_size_limit
-        show_tabs
-        default_tab_index
-        show_metronome
-        show_encoding_controls
-        default_enabled_encodings
-        show_bit_rate_control
-        lock_bit_rate_control
-        default_bit_rate_bits_per_second
-        show_chunk_size_control
-        lock_chunk_size_control
-        default_chunk_size_bits
-        show_router_bandwidth_control
-        default_router_bandwidth
-        show_router_memory_control
-        default_router_memory
-        show_dns_mode_control
-        default_dns_mode
-        input_output_table
-        complete_on_success_condition_not_goals
-      ).map{ |x| x.include?(':') ? x.split(':') : [x,x.camelize(:lower)]}]
-          .each do |dashboard, blockly|
-        # Select value from properties json
-        # Don't override existing valid (non-nil/empty) values
-        property = level_prop[dashboard].presence
-        value = JSONValue.value(level_prop[blockly] || property)
-        level_prop[blockly] = value unless value.nil? # make sure we convert false
-      end
-
```

Comment:
It still sort of exists here, but we don't have to explicitly specify keys anymore: https://github.com/code-dot-org/code-dot-org/pull/2164/files#diff-7c558db0bac67f17752ae9f2a385c24fR166